### DISCUSSION TITLE
Creation of forms for student tracking by teacher

### DISCUSSION THREAD

**Discussion by AntoVJustin** [Sunbird-Ed/Community#16] [Feature request]
I want to be able to create a form in Sunbird ED which the tutor (of a particular classroom) can access and update periodically to track the student's performance. Would it be possible to create a form that looks like this 

![Weekly Reflection Format](https://user-images.githubusercontent.com/104202143/171340441-2bff9ccf-c3f0-4ac6-a431-dea22fc13c9e.png)

**Comment by luckynvdu** [comment]
@AntoVJustin Thanks for sharing the requirement. Would you help with the following -
1. Are you using Sunbird ED or planning to use it? If yes then for which persona and use cases?
2. Are you planning to contribute this feature to Sunbird ED?

Can you also elaborate on the use case, flow, persona, and outcome of this feature? which helps to understand the exact ask
